### DISCUSSION TITLE
Utiliser le conditionnel sur la page des résultats

### DISCUSSION THREAD

**Discussion by JulienWilhelm** [cnumr/EcoIndex#170] [General]
Dans "Comment améliorer mon score ?", il y a les titres suivants :
- La page est trop lourde
- La page est trop complexe
- Il y a trop de requêtes

Leur affichage ne dépend pas des résultats, aussi cela pourrait dérouter certains utilisateurs quand ils obtiennent un "bon score", et se retrouvent ensuite avec une affirmation qui laisse entendre le contraire.

Je propose :
- **Si** la page est trop lourde
- **Si** la page est trop complexe
- **S'il** y a trop de requêtes

Qu'en dites-vous ?

**Comment by vvatelot** [comment]
Ça me semble tout à fait pertinent ! :)

**Comment by aureliebaton** [comment]
Je suis d'accord, go pour le changement !

**Comment by JulienWilhelm** [comment]
J'en prends note !

**Reply by JulienWilhelm** [reply]
C'est fait !
https://github.com/cnumr/EcoIndex/commit/1cee1430a24193b800b9d3d3f328013324f1e0ca
